modules/internal/ChapelBase.chpl

```diff
@@ -1032,20 +1032,23 @@ module ChapelBase {
     return if x != 0i then true else false;
 
   pragma "dont disable remote value forwarding"
+  pragma "no copy return"
```

Comment:
I'm not very familiar with _createFieldDefault. Can you explain why it needs "no copy return" ?

It seems unlikely to me that the param and nilType versions could ever be returning an array view.